demos/test_all.py

```diff
@@ -23,5 +22,4 @@ def test_benchmark(benchmark):
 
     pd.testing.assert_series_equal(df[["u_rms", "nu_top"]], expected, check_names=False)
 
-    if df.name != expected.name:
-        warnings.warn(f"Convergence changed: expected {expected.name}, got {df.name}")
+    assert abs(df.name - expected.name) <= 2
```

Comment:
What's going here? What is df.name and why is it a value that is allowed to be 2 different from expected?

Reply:
Yeah I'm not much of a pandas user... The "name" of the series is its row number, which ends up being the iteration. So we want to (by @drhodrid's suggestion) make sure we're within 2 of the expected number of iterations to convergence. I guess extra documentation would be good here!

Reply:
Same here - I find their API rather ugly. Yes, makes sense - a comment would be good, yeah